doc/classes/EditorInspector.xml

```diff
@@ -20,12 +20,25 @@
 				Returns the object currently selected in this inspector.
 			</description>
 		</method>
+		<method name="get_property_clipboard" qualifiers="const">
+			<return type="Variant" />
+			<description>
+				Get the value currently in the property clipboard.
+			</description>
+		</method>
 		<method name="get_selected_path" qualifiers="const">
 			<return type="String" />
 			<description>
 				Gets the path of the currently selected property.
 			</description>
 		</method>
+		<method name="set_property_clipboard">
+			<return type="void" />
+			<param index="0" name="value" type="Variant" />
+			<description>
+				Put a value into the property clipboard.
```

Comment:
```suggestion
				Assign a value to the property clipboard.
```
Or similar, to make it clear this is not putting a value on some stack but replaces the value, not exactly sure about the wording

Reply:
I can mirror the language of the `DisplayServer::clipboard_set` function?

`Set the property clipboard's content to the given value.`

Reply:
Perfect!